scripts/import/README.md

```diff
@@ -0,0 +1,52 @@
+# Import snapshots
+
+Import snapshots history from git to MongoDB.
+
+The script works by reading snapshots commits from a source repository and insert them in a MongoDB database.
```

Comment:
Does it really? Or  does it read the contents from GitHub and will fail if the repository is not hosted publicly on GitHub?